ITEM.tsv

```diff
@@ -398,206 +398,206 @@ ITEM_20150317_000397	$Viper
+ITEM_20150317_000559	$Aras' Iron Club 
+ITEM_20150317_000560	$Craftable by clerics, mages, alchemists, and even blacksmiths. This club can easily be enchanted with magic. 
+ITEM_20150317_000561	$Smith Mace 
+ITEM_20150317_000562	$The power of this weapon is completely independent of its user's ability or faith. 
+ITEM_20150317_000563	$Klavis Mace 
+ITEM_20150317_000564	$Don't be disappointed that there were no iconic figures in history who used this weapon. You might become the first. 
+ITEM_20150317_000565	$Dio Mace 
+ITEM_20150317_000566	$Thresh Mace 
+ITEM_20150317_000567	$Sastas Mace 
+ITEM_20150317_000568	$Drat Mace 
+ITEM_20150317_000569	$Aston Mace 
+ITEM_20150317_000570	$Devi Mace 
+ITEM_20150317_000571	$Prima Mace 
+ITEM_20150317_000572	$Holy Smasher 
+ITEM_20150317_000573	$Because Clerics are confident in their own faith, the recipe for this weapon is often found, ironically, at unexpected locations. 
+ITEM_20150317_000574	$Kaloo Hammer 
+ITEM_20150317_000575	$New possibilities have been discovered for weaponcrafting thanks to the emergence of new monsters since the Great Plant Cataclysm.   
+ITEM_20150317_000576	$Drake Tail 
+ITEM_20150317_000577	$It is said that the inventor of this weapon lived long before the founding period of the Kingdom. It is incredible that its recipe has survived to this date. 
+ITEM_20150317_000578	$Royal Mace 
+ITEM_20150317_000579	$This weapon used to be only obtainable by Clerics with permission from the royal household. Such restrictions have become meaningless since the Great Plant Cataclysm. 
+ITEM_20150317_000580	$Vubbe Morning Star 
+ITEM_20150317_000581	$ - 3% chance to cause 3 second Stun when attacking. 
+ITEM_20150317_000582	$Some Vubbes do carry useful weapons. 
+ITEM_20150317_000583	$Deathweaver Club 
+ITEM_20150317_000584	$A moment of silence for the previous owner who became the sacrifice required to make this Deathweaver. 
+ITEM_20150317_000585	$Chapparition Mace 
+ITEM_20150317_000586	$No matter what monster you face, this weapon will leave a dent. 
+ITEM_20150317_000587	$Stunner 
+ITEM_20150317_000588	$Some Clerics prefer to disorient their target through means other than preaching. 
+ITEM_20150317_000589	$Valia 
+ITEM_20150317_000590	$It is consecrated with the power of the Goddess during the crafting process. Elder Demons can't stand the sight of this weapon. 
+ITEM_20150317_000591	$Spearfish Rod 
+ITEM_20150317_000592	$May it produce barbed dicipline. 
+ITEM_20150317_000593	$Wooden Shield 
+ITEM_20150317_000594	$Though made of wood, it has been crafted to meet the minimum requirements of a shield. 
+ITEM_20150317_000595	$Wooden Buckler 
+ITEM_20150317_000596	$Surprisingly, the reinforcing of the buckler with a metal rim is a fairly recent development. Historians presume this trend first started around the founding period of the Kingdom. 
```

Comment:
The original line was correct here.  If it's going to change in this way, it would make more sense as:
"Surprisingly, reinforcing bucklers with a metal rim is a fairly recent development."
or
"Surprisingly, the reinforcing of a buckler with a metal rim is a fairly recent development."

I prefer the first choice.